src/main/java/Main.java

```diff
@@ -0,0 +1,67 @@
+import db.Storage;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import operations.OperationHandler;
+import operations.impl.BalanceOperationHandler;
+import operations.impl.PurchaseOperationHandler;
+import operations.impl.ReturnOperationHandler;
+import operations.impl.SupplyOperationHandler;
+import service.DataProcessorService;
+import service.FileReaderService;
+import service.OperationStrategy;
+import service.ReportGeneratorService;
+import service.ReportWriterService;
+import service.impl.DataProcessorServiceImpl;
+import service.impl.FileReaderServiceImpl;
+import service.impl.OperationStrategyImpl;
+import service.impl.ReportGeneratorServiceImpl;
+import service.impl.ReportWriterServiceImpl;
+
+public class Main {
+    public static void main(String[] args) {
+
+        // Ініціалізуємо storage для збереження даних про фрукти та
+        Storage storage = new Storage();
+
+        Map<FruitTransaction.Operation,OperationHandler> operationOperationHandlerMap =
+                new HashMap<>();
+
+        operationOperationHandlerMap.put(FruitTransaction.Operation.BALANCE,
+                new BalanceOperationHandler());
+        operationOperationHandlerMap.put(FruitTransaction.Operation.PURCHASE,
+                new PurchaseOperationHandler());
+        operationOperationHandlerMap.put(FruitTransaction.Operation.RETURN,
+                new ReturnOperationHandler());
+        operationOperationHandlerMap.put(FruitTransaction.Operation.SUPPLY,
+                new SupplyOperationHandler());
+        // Ініціалізуємо об'єкт для читання даних з файлу
+        FileReaderService fileReaderService =
+                new FileReaderServiceImpl("src/main/resources/input.csv");
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)